charts/pod-director/templates/configmap.yaml

```diff
@@ -1,3 +1,22 @@
+{{- $labelKeyRegex := "^([a-z0-9]([-a-z0-9]*[a-z0-9])?\\.)*([a-z0-9]([-a-z0-9]*[a-z0-9])?\\/)?[a-z0-9]([-a-z0-9]*[a-z0-9])?$" }}
+{{- $groupLabel := .Values.config.groupLabel }}
+{{- if $groupLabel }}
+  {{- if gt (len $groupLabel) 63 }}
+    {{- fail (printf "The group label %s is longer than the maximum length 63 for labels" $groupLabel) }}
+  {{- end }}
+  {{- if not (mustRegexMatch $labelKeyRegex $groupLabel) }}
+    {{- fail (printf "The group label %s is not a valid label" $groupLabel) }}
```

Comment:
Add quotes for `%s` to be more visible in the error message the value of $groupLabel